tsl/test/expected/cagg_repair.out

```diff
@@ -311,4 +311,46 @@ UNION ALL
   WHERE conditions."time" >= COALESCE(_timescaledb_internal.to_timestamp(_timescaledb_internal.cagg_watermark(3)), '-infinity'::timestamp with time zone)
   GROUP BY (time_bucket('@ 7 days'::interval, conditions."time"));
 
+-- Tests with old cagg format
+DROP MATERIALIZED VIEW conditions_summary_nojoin;
+CREATE MATERIALIZED VIEW conditions_summary_nojoin
+WITH (timescaledb.continuous, timescaledb.finalized=false) AS
+SELECT
+    time_bucket(INTERVAL '1 week', "time") AS bucket,
+    MIN(temperature),
+    MAX(temperature),
+    SUM(temperature)
+FROM
+    conditions
+GROUP BY
+    1
+WITH NO DATA;
+-- Should rebuild without forcing
+CALL _timescaledb_internal.cagg_try_repair('conditions_summary_nojoin', FALSE);
+DEBUG:  [cagg_rebuild_view_definition] public.conditions_summary_nojoin has being rebuilded!
+\d+ conditions_summary_nojoin
```

Comment:
This error message doesn't seem right, being->been. Maybe should have mentioned in the earlier PR.

Reply:
Fixed, thanks!